### PR TITLE
Add Java to the list of not supported languages in 'Sequence'

### PR DESCRIPTION
**Pull request by matiasroldan6**
… localizations excepting English and Spanish

### CODE REVIEW

File: Specs/Sequence/Dutch/PhoneNumberModel.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [
   {
     "Input": "Tel: (+31)12-3456789.",
-    "NotSupported":"javascript",
+    "NotSupported":"javascript,java",
```
Comment (Batta32) [inline Specs/Sequence/Dutch/PhoneNumberModel.json]:
```suggestion
    "NotSupported":"javascript, java",
```

File: Specs/Sequence/German/PhoneNumberModel.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [
   {
     "Input": "Tel: (+31)12-3456789.",
-    "NotSupported":"python, javascript",
+    "NotSupported":"python, javascript,java",
```
Comment (Batta32) [inline Specs/Sequence/German/PhoneNumberModel.json]:
```suggestion
    "NotSupported":"python, javascript, java",
```

File: Specs/Sequence/Hindi/PhoneNumberModel.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 [
   {
     "Input": "Tel: (+31)12-3456789.",
-    "NotSupported":"python, javascript",
+    "NotSupported":"python, javascript,java",
```
Comment (Batta32) [inline Specs/Sequence/Hindi/PhoneNumberModel.json]:
```suggestion
    "NotSupported":"python, javascript, java",
```

File: Specs/Sequence/Italian/PhoneNumberModel.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
 [
   {
     "Input": "Tel: (+31)12-3456789.",
-    "NotSupported":"javascript",
+    "NotSupported":"javascript,java",
```
Comment (Batta32) [inline Specs/Sequence/Italian/PhoneNumberModel.json]:
```suggestion
    "NotSupported":"javascript, java",
```

File: Specs/Sequence/Turkish/PhoneNumberModel.json [hunk 5]
```diff
@@ -1,7 +1,7 @@
 [
   {
     "Input": "Tel: (+31)12-3456789.",
-    "NotSupported":"javascript",
+    "NotSupported":"javascript,java",
```
Comment (Batta32) [inline Specs/Sequence/Turkish/PhoneNumberModel.json]:
```suggestion
    "NotSupported":"javascript, java",
```

**Review by Batta32 — CHANGES_REQUESTED** [review]
Minor changes! Apart from that LGTM

**Review by fran893 — APPROVED** [review]

**Review by Batta32 — APPROVED** [review]
LGTM

**Pull request merged**